std/stdio.d

```diff
@@ -346,8 +346,8 @@ Hello, Jimmy!
  */
 struct File
 {
-    import std.traits : isScalarType, isArray;
```

Comment:
I know you want to alphabetize them but when you're not modifying the import itself there's no meaningful change; this is just churn